back-end/src/auth/utils/auth.cookie.ts

```diff
@@ -3,19 +3,19 @@ import cookie from 'cookie';
 export const createAuthCookie = (strategy: string) => {
 	return cookie.serialize('token', strategy, {
 		httpOnly: true,
-		secure: false, // config.jwt.cookie.secure
+		secure: true, // config.jwt.cookie.secure
```

Comment:
This may cause some issues with some users, so we will need to be careful about it.